src/js/redux/reducers/account/accountReducer.js

```diff
@@ -42,7 +42,8 @@ export const initialState = {
             appropriations: 0
         }
     },
-    totalSpending: 0
+    totalSpending: 0,
+    dataAsOf: null // will be a moment obj
```

Comment:
Is there any reason we are using `dataAsOf` instead of latest submission?

Do we really want to store this in account considering we are using it outside of account?

Reply:
(1) Yes. `dataAsOf`  is what you see the in the UI. It is also more descriptive of how the data is actually used in the UI. It is not only used to show the latest submission period but also the active FY.

(2) Yes. I decided to put this property here because it represents the latest **account** specific data. Our redux store is organized on a page-by-page basis. Each parent level key space is for a specific page in the application. This constitutes a departure from that pattern as we now have account level data (latestSubmission/dataAsOf) that is used on the SpendingExplorer, Federal Accounts Landing and Detail Page, Download Page, and in the future the Agency V2 Page. Creating a new global redux key-space for this data appears to me to be overkill. We would have to create a new reducer, a new file, etc...